tasks/pipeline-dbconf.yml

```diff
@@ -36,6 +36,7 @@
     encoding: "utf8"
     collation: "utf8_unicode_ci"
   register: "createdb_result"
+  when: 'archivematica_src_am_database_host == "localhost"'
```

Comment:
What happens when the database host is not localhost? Does the database and user need to be created manually?